docs/release-notes/cloud/cloud-2023-07.md

```diff
@@ -0,0 +1,90 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud July 2023.
+included_jira_versions: ['2023.07']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/7.5.46
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/7.6.39
+---
+
+# Cloud July 2023
+
+These release notes are for the Codacy Cloud updates during July 2023.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Added support for AI-generated context-aware comments and fix suggestions directly in your Git provider. For more details, [see the announcement on our blog](https://blog.codacy.com/codacy-quality-ai-actionable-fixes/). (CY-6855, HRZ-286)
+
+    ![AI-generated context-aware comments and fix suggestions](../images/cy-6855.png)
+
+-   You can now use [Security and risk management](../../organizations/managing-security-and-risk.md#managing-integrations) to detect security vulnerabilities across your Codacy repositories and Jira issues and assign the Security Manager role to any organization member. For more details, [see the announcement on our blog](https://blog.codacy.com/centralized-view-of-security-issues/). (HRZ-6, PUL-1924, HRZ-180)
```

Comment:
@nicklem As we are announcing the feature for the first time, should we link to the feature page instead of a specific section?

```suggestion
-   You can now use [Security and risk management](../../organizations/managing-security-and-risk.md) to detect security vulnerabilities across your Codacy repositories and Jira issues and assign the Security Manager role to any organization member. For more details, [see the announcement on our blog](https://blog.codacy.com/centralized-view-of-security-issues/). (HRZ-6, PUL-1924, HRZ-180)
```

Reply:
Absolutely! Good catch. This was an earlier fragment link when the entry only mentioned the Jira integration.